server/.env

```diff
@@ -1,4 +1,7 @@
 TEAM_NAME="Jason Chui, Philippe Charpentier, Ziqi Shen, Milan Verma"
 FLASK_ENV="development"
 MONGODB_PWD="replace with DBAdminPWD provided on slack"
-TEST_ENV="Hello Team BLT"
\ No newline at end of file
+TEST_ENV="Hello Team BLT"
+SECRET_KEY = 'girlsAreDangerous'
+MONGODB_HOST = "mongodb+srv://TeamBLT-Admin:jcPCzsMV2021HatchwaysTeamBLT@teambltcluster0.5hi8c.mongodb.net/test?retryWrites=true&w=majority"
+JWT_SECRET_KEY = 'happyValentine'
```

Comment:
This environment file should be included in .gitignore so it isn't committed to the repository.